action.yml

```
@@ -14,6 +14,13 @@ inputs:
   refresh-message-position:
     description: "If a message with the same id, this option allow to refresh the position of the message to be the last one posted."
     default: "false"
+  repo-owner:
+    description: "The repo owner."
+    default: "${{ github.repository_owner }}"
+    required: false
```

```suggestion
    required: true
```

Since you're setting a default you can mark this as required.